components/plurals/tests/fixtures/mod.rs

```diff
@@ -121,3 +121,49 @@ pub enum RuleTestOutput {
 
 #[derive(Deserialize)]
 pub struct RuleTestSet(pub Vec<RuleTest>);
+
+#[derive(Clone, Copy, Debug, Deserialize)]
+pub enum PluralRuleTypeInput {
+    Cardinal,
+    Ordinal,
+}
+
+impl From<PluralRuleTypeInput> for PluralRuleType {
+    fn from(other: PluralRuleTypeInput) -> Self {
+        match other {
+            PluralRuleTypeInput::Cardinal => PluralRuleType::Cardinal,
+            PluralRuleTypeInput::Ordinal => PluralRuleType::Ordinal,
+        }
+    }
+}
+
+#[derive(Debug, Deserialize)]
+pub enum PluralCategoryInput {
+    Zero,
+    One,
+    Two,
+    Few,
+    Many,
+    Other,
+}
```

Comment:
Nit (optional): Plural rule category strings should be lowercase.  I know this is only for a test fixture, but I think it may be cleaner if you had "zero" and "one" in your JSON files instead of "Zero" and "One".